_stbt/ocr.py

```diff
@@ -65,6 +65,9 @@ class OcrMode(IntEnum):
     SINGLE_WORD = 8
     SINGLE_WORD_IN_A_CIRCLE = 9
     SINGLE_CHARACTER = 10
+    SPARSE_TEXT = 11
```

Comment:
This should be useful. I'd expect most of our users will want `SINGLE_UNIFORM_BLOCK_OF_TEXT` for `stbt.ocr` and `SPARSE_TEXT` for `stbt.match_text`.